contents/blog/product-engineer-vs-software-engineer.md

```diff
@@ -0,0 +1,66 @@
+---
+date: 2022-12-01
+title: "Product engineer vs software engineer: what's the difference?"
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+author: ["ian-vanagas"]
+featuredImage: ../images/blog/posthog-engineering-blog.png
+featuredImageType: full
+categories: ["Startups", "Guides"]
+---
+
+There are always arguments about what is the best way to run a tech company, how you should structure a company, and what roles you should have. Tech companies are endlessly trying to innovate in company composition to get work done effectively.
+
+A recent addition to the debate is the role of the product engineer. The role of the product engineer changed the way companies think about engineering. Is a greater focus on product necessary to succeed now? Are older structures and engineering roles outdated?
+
+To help you think through this yourself, we’ll compare the role of product engineer with the role of software engineer.
+
+## What does each role focus on?
+
```

Comment:
Here's the TL;DR: 
- Product engineers focus on building great products.
- Software engineers focus on building great software.

Let's break those points down a bit more.